types/src/chapter.rs

```diff
@@ -0,0 +1,21 @@
+use uuid::Uuid;
+use {
+    crate::*,
+};
+
+#[derive(Debug, Clone)]
+pub struct Chapter {
```

Comment:
Can we either call this `CollectionChapter` or have it live in the `collection` module and use it like `collection::Chapter` (that might be better)? They're very closely associated data and I want to make that clear in the naming.

Reply:
Yes, that makes sense! The Chapter type now lives in the Collection module.